src/views/MainView.js

```diff
@@ -13,12 +13,14 @@ import {API_BASE_URL} from "../services/backend/constants";
 
 const MainView = (props) => {
   const {isLogged, user} = props;
-  let eventSource = null;
-  if (user) {
-    // authenticate user
-    props.dispatch(authenticate(user));
+  // authenticate user
+  if (user) props.dispatch(authenticate(user));
```

Comment:
i think it would be better to move this into `moveEffect` with an empty dependency array
https://reactjs.org/docs/hooks-effect.html